validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkFeedbackRequestValidatorTest.java

```diff
@@ -124,47 +130,40 @@ public class BulkFeedbackRequestValidatorTest extends AbstractValidatorUnitTest
     /**
      * Setup of the Validator and Domain class instance.
      */
-    public void setUpLocalTests() {
+    @BeforeEach
+    @Override
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);

```

Comment:
Is this needed?  I think the @ExtendWith(MockitoExtension.class) annotation initialises the mocked objects.